code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -908,3 +908,17 @@
 			return
 	clear_products(holder, step_volume_added)
 	holder.my_atom.audible_message(span_notice("[icon2html(holder.my_atom, viewers(DEFAULT_MESSAGE_RANGE, src))] The reaction gives out a fizz, teleporting items everywhere!"))
+
+/datum/chemical_reaction/ants
+	results = list(/datum/reagent/ants = 3)
+	required_reagents = list(/datum/reagent/ants = 2, /datum/reagent/consumable/sugar = 6)
+
+/datum/chemical_reaction/big_ant // We're basically glueing ants together with synthflesh & maint sludge to make a bigger ant.
+	required_reagents = list(/datum/reagent/ants = 40, /datum/reagent/medicine/c2/synthflesh = 20, /datum/reagent/drug/maint/sludge = 5)
+	required_temp = 374
+
+/datum/chemical_reaction/big_ant/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = rand(1, created_volume), i <= created_volume, i++)
```

Comment:
I wasn't aware this syntax worked, thought you needed ;
On that topic, you can use `for(var/i in rand(1, created_volume) to created_volume)` to achieve the same effect, tho it really just comes down to a matter of preference

Reply:
I'll be honest that I just copied this bit from corgium, since that had essentially the same effect I was looking for. I'll try out yours though.

Reply:
It's not making anything anymore.

Reply:
actually, it doesn't seem to make anything at all in the first place, which is strange.

Reply:
Nevermind! I figured it out, reagent code was changed to add a bunch of flags that I hadn't added yet.